tests/fixtures/rendering.yml

```diff
@@ -250,31 +250,42 @@
     '#id': file_link
     '#fields':
       files:
-        - title: "Example document 1"
-          mime: "application/pdf"
-          url: "http://example.com/document1.pdf"
+        - title: "Example document"
+          mime: "application/doc"
+          url: "http://example.com/document.doc"
           size: "150000"
-          name: "document.pdf"
-        - title: "Example document 2"
-          mime: "application/pdf"
-          url: "http://example.com/document2.pdf"
+          name: "document.doc"
+        - title: "Example presentation"
+          mime: "application/ppt"
+          url: "http://example.com/presentation.ppt"
           size: "160000"
-          name: "document.pdf"
-        - title: "Example document 3"
-          mime: "application/pdf"
-          url: "http://example.com/document3.pdf"
+          name: "presentation.ppt"
+        - title: "Example video"
+          mime: "application/mpeg"
+          url: "http://example.com/video.mpeg"
           size: "170000"
-          name: "document.pdf"
-  assertions:
-    count:
-      'div.ecl-file__link-container': 3
-      'a[href="http://example.com/document1.pdf"]': 1
-      'a[href="http://example.com/document2.pdf"]': 1
-      'a[href="http://example.com/document3.pdf"]': 1
-    equals:
-      'a[href="http://example.com/document1.pdf"] span.ecl-file__title': "Example document 1"
-      'a[href="http://example.com/document2.pdf"] span.ecl-file__title': "Example document 2"
-      'a[href="http://example.com/document3.pdf"] span.ecl-file__title': "Example document 3"
+          name: "video.mpeg"
+        - title: "Example spreadsheet"
+          mime: "application/xls"
+          url: "http://example.com/spreadsheet.xls"
+          size: "170000"
+          name: "spreadsheet.xls"
+  assertions:
+    count:
+      'div.ecl-file__link-container': 4
+      'a[href="http://example.com/document.doc"]': 1
+      'a[href="http://example.com/presentation.ppt"]': 1
+      'a[href="http://example.com/video.mpeg"]': 1
+      'a[href="http://example.com/spreadsheet.xls"]': 1
+      'span.ecl-icon--file': 1
+      'span.ecl-icon--presentation': 1
+      'span.ecl-icon--video': 1
+      'span.ecl-icon--spreadsheet': 1
+    equals:
+      'a[href="http://example.com/document.doc"] span.ecl-file__title': "Example document"
```

Comment:
If you use this selector:
`a[href="http://example.com/document.doc"] span.ecl-icon--file + span.ecl-file__title`
you can assert also that the correct icon is placed in the correct place.

Reply:
Update the selector also for the other lines.